js-intro.js

```diff
@@ -6,19 +6,35 @@
 var mantra = "Be the dev"
 
 // 1a. Write the code that determines if there is a 'B' in mantra.
-
+console.log("\n1a------------------------")
+console.log(mantra.includes('B'))
 
 
 // 1b. Write the code that determines if there is an 'x' in mantra.
-
+console.log("\n1b------------------------")
+console.log(mantra.includes('x'))
 
 
 // 1c. Write the code that determines if there is a 'v' in mantra.
-
+console.log("\n1c------------------------")
+console.log(mantra.includes('v'))
 
 
 // 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
-
+console.log("\n1d------------------------")
+function returnLetter (string, letter) {
+    if (letter === " " || letter === undefined) {
+        return "Enter a letter."
+    }
+    else if (string.includes(letter) === true) {
+        return "\"" + letter + "\" exists within the string."
+    }
+    else {
+        return "Try a different letter."
+    }
+}
+
+console.log(returnLetter(mantra, "e"))
 
```

Comment:
Nice work - you could incorporate string interpolation to your return statement to avoid the negation of quotes. Just a thought.